src/docs/linea.mdx

```diff
@@ -0,0 +1,192 @@
+            title: 'L2 → L1',
+            rows: [
+                {
+                    label: 'Latency',
+                    tooltip: "The time it takes for a message to be made available on Ethereum after being included in an Linea block and sequenced to L1. Messages can only be claimed after ZK proof has been submitted on L1 that verifies it's been submitted on L2.",
+                    value: '1 to 32 hours'
+                },
+                {
+                    label: 'Cost',
+                    tooltip: 'The gas cost of executing a message on Ethereum after being sent from Linea. The fee for the prover is paid by the sender on L2.',
+                    value: '73 000 L2 gas'
+                },
+                {
+                    label: 'Delivery method',
+                    tooltip: 'How the smart contract on the target chain is called.',
+                    value: 'Manual'
+                },
+            ],
+        },
+    ]} />
+
+</Section>
+
+<Section title="OPCODEs">
+
+    <Legend />
+
+    Linea uses the London version of the EVM.
+
+    The following table encapsulates EVM OPCODE differences of the Rollup compared to the canonical Ethereum L1 implementation.
+
+    | Opcode | Name | Solidity Equivalent | Rollup Behaviour | Ethereum L1 Behaviour |
+    | :----- | :--- | :-------------------| :--------------- | :-------------------- |
+    | 44 | PREVRANDAO (legacy DIFFICULTY) | `block.difficulty` | Returns a fixed number "2" | Get the output of the randomness beacon provided by the beacon chain <Modified /> |
+    | 5F | PUSH0 |  | Not supported yet! <br /><br /> Solidity version `0.8.20` or higher can only be used with an evm version lower than the default `shanghai`. <br /><br /> Versions up to `0.8.19` (included) are fully compatible | Places value 0 on the stack <Unsupported /> |
+
+</Section>
+
+<Section title="Precompiled Contracts">
+
+    > ⚠️ All Precompiled Contacts defined in the canonical Ethereum L1 implementation (until London hard fork) have the same behaviour on the Rollup, however, the execution of those Precompiles is not covered by the circuits and therefore not present in the Validity proof
+
+</Section>
+
+<Section title="System Contracts">
+
+    The Rollup has introduced the following system contracts.
+
+    | Address | Name | Description |
+    | :--- | :--- | :--- |
+    | <Copy value="0xC84832f69bFFbC1A94E44a157A342766E959ED27" label="0xC84...27" /> | <Reference label="PlonkVerifierFull" url="https://github.com/Consensys/linea-contracts/blob/main/contracts/verifiers/PlonkVerifierFull.sol" /> | Plonk verifier contract used by the Linea zkEVM rollup. |
+    | <Copy value="0xb32c3D0dDb0063FfB15E8a50b40cC62230D820B3" label="0xb32...B3" /> | <Reference label="ZkEvmV2" url="https://github.com/Consensys/linea-contracts/blob/main/contracts/ZkEvmV2.sol" /> |The main contract of the Linea zkEVM rollup. Contains state roots, the verifier addresses and manages messages between L1 and the L2.|
```

Comment:
This is the "implementation" contract for the zkEVMv2. The proxy contract is `0xd19d4B5d358258f05D7B411E21A1460D11B0876F`. We can leave the `address` value to the proxy contract as this is the entrypoint and in the `Description` state:

```The main contract of the Linea zkEVM Rollup. Contains the L2 state roots. It handles sequencing and finalisation of L2 blocks and provides messaging functionality between L1 and L2. Provides functionality for sending and receiving messages between domains (Ethereum and Linea). Details on the [L1 Message Service](https://github.com/Consensys/linea-contracts/blob/main/contracts/messageService/l1/L1MessageService.sol)```